package.json

```diff
@@ -142,6 +141,7 @@
     "react-native-patina": "^0.1.6",
     "react-native-permissions": "^4.1.5",
     "react-native-piratechain": "^0.4.16",
+    "react-native-qr-svg": "^1.3.2",
```

Comment:
This adds 1MB of javascript to our code which is pretty sizeable. I talked to William and he agreed this could easily create a perf issue. The icon could just be a View that you overlay on top of the QRcode. No additional library needed.